redaxo/src/core/lib/system_report.php

```diff
@@ -41,8 +41,8 @@ public function get()
         ];
 
         $security = rex_setup::checkPhpSecurity();
-        if ($security) {
-            $data['PHP']['Warning'] = $security;
+        foreach ($security as $warning) {
+            $data['PHP']['Warning'] = $warning .'<br/>';
```

Comment:
Du überschreibst hier in der Schleife immer den selben Wert

Reply:
fixed